docs/topics/getting-started-create-booster.adoc

```diff
@@ -9,6 +9,6 @@ IMPORTANT: This steps requires you to login using Red Hat Developers account.
 . Choose your deployment type.
 . Follow the instructions to create a booster of your choice.
 
-The _Use Continuous Delivery_ option creates a new GitHub repository of your booster and launches it to {OpenShiftOnline} while _Build and Run Locally_ creates a ZIP of your booster that you can download to build and run locally.
+The _Deploy in OpenShift Online_ option creates a new GitHub repository of your booster and launches it to {OpenShiftOnline} while _Build and Run Locally_ creates a ZIP of your booster that you can download to build and run locally.
```

Comment:
s/repository of your/repository containing
s/a ZIP of/a ZIP containing

Reply:
thanks @ALRubinger updated per feedback.